Description: Fix warning when module is used.
 Since a recent upgrade in the depedencies, maybe Object::Pad,
 this module throws a warning on something like `perl -w -MTickit::Widget::FloatBox -e1':
 # "my" variable $self masks earlier declaration in same scope at /usr/share/perl5/Tickit/Widget/FloatBox.pm line 358.
 Let's remove the line.
Origin: vendor
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2020-04-30

--- a/lib/Tickit/Widget/FloatBox.pm
+++ b/lib/Tickit/Widget/FloatBox.pm
@@ -355,7 +355,6 @@
 
 method hide ()
 {
-   my $self = shift;
    $_hidden = 1;
 
    $_child->window->hide if $_child->window;
